calibration/step-3-postprocess-measure.py

```diff
@@ -0,0 +1,122 @@
+###############################################################################
+# Copyright (C) 2024 Habana Labs, Ltd. an Intel Company
+###############################################################################
+import argparse
+import json
+import os
+import sys
+
+import numpy as np
+
+
+def fix_cache_inputs(json_data, args):
+    layer_indexes = set([int(key.split('.')[2]) for key in json_data['Nodes'] if key.startswith('model.layers.')])
+    for layer_index in range(len(layer_indexes)):
+        matmul_av_input = None
+        v_cache_input = None
+        matmul_qk_input = None
+        k_cache_input = None
+
+        attn_name = "attn"
+        k_cache_name = "k_cache"
+        v_cache_name = "v_cache"
+        if args.deepseek:
+            attn_name = "mla_attn"
```

Comment:
the mla attn keys changed in plugin. Changing to mla_attn.mla_attn worked for me. Here are example keys:

model.layers.0.self_attn.mla_attn.mla_attn.impl.matmul_av
model.layers.0.self_attn.mla_attn.mla_attn.impl.matmul_qk
model.layers.0.self_attn.mla_attn.mla_attn.impl.latent_cache_k